### PR TITLE
Post PR #91 Cleanup

### PR DESCRIPTION
**Pull request by colincasey**
We can remove these separated workflows now that the following PRs have been merged:

- https://github.com/heroku/buildpacks-jvm/pull/554
- https://github.com/heroku/buildpacks-php/pull/14
- https://github.com/heroku/buildpacks-go/pull/123
- https://github.com/heroku/buildpacks-ruby/pull/183
- https://github.com/heroku/buildpacks-nodejs/pull/609
- https://github.com/heroku/buildpacks-python/pull/58
- https://github.com/heroku/procfile-cnb/pull/156

### CODE REVIEW

File: .github/workflows/_buildpacks-release-package.yml [hunk 1]
```diff
@@ -1,83 +0,0 @@
...
-        run: pack buildpack package ${{ inputs.buildpack_artifact_prefix }}.cnb --config ${{ steps.libcnb-package.outputs.output_dir }}/package.toml --format file -v
-
-      - name: Create Docker Image
-        run: pack buildpack package ${{ inputs.buildpack_id }} --config ${{ steps.libcnb-package.outputs.output_dir }}/package.toml -v
-
-      - name: Save Docker Image
-        run: docker save ${{ inputs.buildpack_id }} | zstd -T0 --long=31 -o ${{ inputs.buildpack_artifact_prefix }}.tar.zst
-
-      - name: Store buildpack release artifacts
-        uses: heroku/languages-github-actions/.github/actions/save-buildpack-release@main
```
Comment (edmorley) [inline .github/workflows/_buildpacks-release-package.yml:78]: With this removal, `.github/actions/save-buildpack-release` is now unused and can be deleted too :-)

File: .github/workflows/_buildpacks-release-publish-docker.yml [hunk 2]
```diff
@@ -1,65 +0,0 @@
...
-        description: The token to login to Docker Hub with
-
-jobs:
-  release-publish-docker:
-    name: Publish → Docker
-    runs-on: ubuntu-latest
-    steps:
-      - name: Restore buildpack release artifacts
-        id: artifacts
-        uses: heroku/languages-github-actions/.github/actions/restore-buildpack-release@main
```
Comment (edmorley) [inline .github/workflows/_buildpacks-release-publish-docker.yml:42]: Similarly, with this removal, `.github/actions/restore-buildpack-release` is now unused and can be deleted too :-)

File: .github/workflows/_buildpacks-release-detect.yml [hunk 3]
```diff
@@ -1,26 +0,0 @@
-name: _buildpacks-release-detect
```
Comment (edmorley) [inline .github/workflows/_buildpacks-release-detect.yml:1]: The references to the old workflows can now be removed from the README:
https://github.com/heroku/languages-github-actions#workflows

(Also happy for the README change to wait until another PR, if you'd prefer to document the new workflow at the same time as removing the old ones, and don't want to add it in this PR)

**Review by edmorley — APPROVED** [review]

File: .github/workflows/_buildpacks-release-detect.yml (hunk 3 above)
Comment (colincasey) [reply .github/workflows/_buildpacks-release-detect.yml:1]: I was thinking of updating the docs as a separate PR but it makes more sense to include them with these removals.

**Pull request merged**